lib/pages/login_page.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:ui';
+
+import 'package:email_validator/email_validator.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+class LoginPage extends StatefulWidget {
+  LoginPage({Key key}) : super(key: key);
+
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Center(
+        child: Container(
```

Comment:
Optional: We can use `Stack` widget here 😄

Reply:
I tried Stack, but found this weird issue (simply replace Container with Stack and add the rest as the children): there is a weird space at the end 😳 
![weird](https://user-images.githubusercontent.com/13435717/112297465-848b4680-8cc8-11eb-9afc-6c03b9e79619.gif)

Reply:
oh, I have tried, no issue at all 😂 

https://user-images.githubusercontent.com/60863885/112304850-8fe27000-8cd0-11eb-9777-f733cd78685c.mp4

```dart
Scaffold(
      resizeToAvoidBottomInset: false,
      body: Stack(
        children: [
          Container(
            constraints: BoxConstraints.expand(),
            decoration: BoxDecoration(
              image: DecorationImage(
                  fit: BoxFit.cover,
                  image: AssetImage('assets/images/sign_in_background.png')),
            ),
          ),
          BackdropFilter(
            filter: ImageFilter.blur(sigmaX: 12, sigmaY: 12),
            child: Container(
              color: Colors.white10,
            ),
          ),
          Padding(
            padding: EdgeInsets.only(left: 24, right: 24),
            child: Column(
              // mainAxisAlignment: MainAxisAlignment.start,
              crossAxisAlignment: CrossAxisAlignment.stretch,
              children: [
                Container(
                  margin: const EdgeInsets.only(top: 150.0),
                  child: SvgPicture.asset('assets/images/logo.svg'),
                ),
                SizedBox(height: 100),
                TextFormField(
                  keyboardType: TextInputType.emailAddress,
                  decoration: InputDecoration(
                      labelStyle: TextStyle(color: Colors.white30),
                      floatingLabelBehavior: FloatingLabelBehavior.never,
                      border: OutlineInputBorder(
                          borderSide: BorderSide.none,
                          borderRadius: BorderRadius.circular(12.0)),
                      fillColor: Colors.white24,
                      filled: true,
                      labelText: 'Email'),
                  style: TextStyle(color: Colors.white),
                  validator: (value) {
                    if (value == null || value.isEmpty) {
                      return "Please enter your email";
                    } else if (!EmailValidator.validate(value)) {
                      return "Please enter a valid email";
                    }
                    return null;
                  },
                ),
                SizedBox(height: 20),
                TextFormField(
                  keyboardType: TextInputType.text,
                  decoration: InputDecoration(
                      labelStyle: TextStyle(color: Colors.white30),
                      floatingLabelBehavior: FloatingLabelBehavior.never,
                      border: OutlineInputBorder(
                          borderSide: BorderSide.none,
                          borderRadius: BorderRadius.circular(12.0)),
                      fillColor: Colors.white24,
                      filled: true,
                      labelText: 'Password'),
                  obscureText: true,
                  style: TextStyle(color: Colors.white),
                ),
                SizedBox(height: 50),
                ElevatedButton(child: Text('Log in'), onPressed: () {})
              ],
            ),
          )
        ],
      ),
    );
  }
```

Reply:
@manh-t but that means Stack introduces another view layer 🤔, would it affect the performance? 🤔 I thought your suggestion was to replace container with Stack + Image + ...(remove container)